curation-authors-backfill/externalCaller/curatedCorpusApiCaller.ts

```diff
@@ -0,0 +1,43 @@
+import { CorpusInput } from '../types';
+import config from '../config';
+import fetch from 'node-fetch';
+import { generateJwt } from '../jwt';
+import { getCurationToolsDataSyncPrivateKey } from '../secretManager';
+
+/**
+ * Call the importApprovedCorpusItem mutation
+ * @param data input variables for the mutation. Of the type CorpusInput
+ * @returns json response of type ImportApprovedCorpusItemPayload
+ */
+export async function importApprovedCorpusItem(data: CorpusInput) {
+  const mutation = `
+  mutation importApprovedItem($data: ImportApprovedCorpusItemInput!) {
```

Comment:
did we add the `authors` field as a part of mutation input?

Reply:
looks like we need to use `UpdateApprovedCorpusItemAuthors` mutation for updating authors https://github.com/Pocket/curated-corpus-api/blob/main/schema-admin.graphql#L724